TriforkSwiftExtensions/TriforkSwiftExtensions/UIView/UIViewExtension.swift

```diff
@@ -113,6 +119,18 @@ public extension UIView {
     func findFirstResponder() -> UIView? {
         return findFirstResponder(inView: self)
     }
+
+    /// Adds `view` as subview to `self` and sets constraints for all edges with specified inset.
+    func addSubview(_ view: UIView, withEdgeInsets inset: UIEdgeInsets) {
+        view.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(view)
+        NSLayoutConstraint.activate([
+            view.leadingAnchor.constraint(equalTo: leadingAnchor, constant: inset.left),
```

Comment:
Maybe we should comment that it does not use the margin here? 
There is a huge differens.

Reply:
Personally, I never use the margins - but maybe I should? 😅
Maybe we should have both versions? `addSubviewWithMargins` ? What do you think?

Reply:
Yeah, there is something about those margins and the stuff we talked about yesterday. 
But let's keep this and see if we need the other one.